test/integration/test-video-manager.js

```diff
@@ -114,8 +114,8 @@ describe.configure().ifChrome().skipOldChrome().run('VideoManager', function() {
 
     });
 
-    it.configure().ifChrome().skipOldChrome().run(`autoplay - autoplay not
-        supported should behave like manual play`, () => {
+    it.skip(`autoplay - autoplay not supported should behave
```

Comment:
Please add `TODO(aghassemi)` for these since Ali gracefully volunteered to look.